src/main/java/DBDriverWrapper.java

```diff
@@ -0,0 +1,13 @@
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.apache.arrow.vector.VectorSchemaRoot;
+
+public interface DBDriverWrapper {
+
+  Connection connect(String host, String database, String userName, String password);
+
+  VectorSchemaRoot executeQuery(String sqlQuery) throws SQLException;
+
+  Connection getConnection();
```

Comment:
Should not expose the connection in the interface, this allows other code to manipulate the connection